2022-06-11 06:25:30,106 INFO 22929 [egg-logrotator] agent logger reload: got log-reload message
2022-06-11 07:45:05,278 ERROR 22929 nodejs.ClusterClientNoResponseError: client no response in 4755064ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/talk/Desktop/document/rio/exchange/portal/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 22929
hostname: talkdeMacBook-Pro.local

2022-06-11 10:01:57,124 ERROR 22929 nodejs.ClusterClientNoResponseError: client no response in 8211089ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/talk/Desktop/document/rio/exchange/portal/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 22929
hostname: talkdeMacBook-Pro.local

2022-06-11 22:51:24,319 ERROR 22929 nodejs.ClusterClientNoResponseError: client no response in 43266657ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/talk/Desktop/document/rio/exchange/portal/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 22929
hostname: talkdeMacBook-Pro.local

2022-06-11 23:30:04,338 ERROR 22929 nodejs.ClusterClientNoResponseError: client no response in 2319236ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/talk/Desktop/document/rio/exchange/portal/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 22929
hostname: talkdeMacBook-Pro.local

2022-06-12 01:22:13,747 ERROR 22929 nodejs.ClusterClientNoResponseError: client no response in 6728215ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/talk/Desktop/document/rio/exchange/portal/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 22929
hostname: talkdeMacBook-Pro.local

